File: src/main/java/com/fuelproject/data/Fuel.java
package com.fuelproject.data;

import java.io.Serializable;

public class Fuel implements Serializable {

    public long fuelId;
    public String name;
    public Float price;

    @Override
    public String toString() {
        return "Fuel{" +
            "fuelId=" + fuelId +
            ", price=" + price +
            ", name=" + name +
            '}';
    }
}

File: src/main/java/com/fuelproject/data/Fuel.kt
package com.fuelproject.data

import java.io.Serializable

class Fuel : Serializable {
    var fuelId: Long = 0
    var name: String? = null
    var price: Float? = null
    override fun toString(): String {
        return "Fuel{" +
                "fuelId=" + fuelId +
                ", price=" + price +
                ", name=" + name +
                '}'
    }
}
